packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -60,7 +85,24 @@ export const initializeNodeEnvironment: EnvironmentInitializer<
     child.stdout?.setEncoding('utf8');
     child.stderr?.setEncoding('utf8');
     child.stdout?.on('data', console.log); // eslint-disable-line no-console
-    child.stderr?.on('data', console.error); // eslint-disable-line no-console
+
+    let lastErrors: string[] = [];
+    let lastErrorDateStamp: number | undefined = undefined;
```

Comment:
DateStamp - it's called timestamp, a single word.

Reply:
renamed to timestamp and refactored